src/pages/AdminSpaceSettings/AdminSpaceSettings.tsx

```diff
@@ -19,32 +18,32 @@ import { LoadingPage } from "components/molecules/LoadingPage";
 
 import { AdminRestricted } from "components/atoms/AdminRestricted";
 
-import "./AdminAdvancedSettings.scss";
+import "./AdminSpaceSettings.scss";
 
-export enum AdminAdvancedTab {
+export enum AdminSpaceSettingsTab {
   basicInfo = "basic-info",
   entranceExperience = "entrance-experience",
-  advancedMapSettings = "advanced-map-settings",
 }
 
-export interface AdminAdvancedSettingsRouteParams {
+export interface AdminSpaceSettingsRouteParams {
   venueId?: string;
-  selectedTab?: AdminAdvancedTab;
+  selectedTab?: AdminSpaceSettingsTab;
 }
 
-const adminAdvancedTabLabelMap: Readonly<Record<AdminAdvancedTab, String>> = {
-  [AdminAdvancedTab.basicInfo]: "Start",
-  [AdminAdvancedTab.entranceExperience]: "Entrance",
-  [AdminAdvancedTab.advancedMapSettings]: "Advanced",
+const adminAdvancedTabLabelMap: Readonly<
```

Comment:
[NOTE]

This can probably be safe to extract in a settings file under `./src/settings/` along with similar Admin UX navigation constants, in some future PR once few more constants of this kind are identified